src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,28 @@
+package bit.javaoop;
+
+public class Company {
+    String name;
+    CEO ceo;
+
+    public Company(String name, CEO ceo) {
+        this.name = name;
+        this.ceo = ceo;
+    }
+
+
+    public String getCeo() {
+        return ceo.name;
+    }
+
+    @Override
+    public String toString() {
+        String result = this.getCeo() + " - CEO\n";
+        for (Manager manager : ceo.managerList) {
+            result += "\t" + manager.getName() + " - Manager\n";
+            for (AbstractEmployee employee : manager.getMyStaffList()) {
+                result += "\t\t" + employee.getName() + " - Employee\n";
```

Comment:
Takie konkatenacje w pętli powinno się robić za pomocą StringBuildera na zajęciach opowiemy dlaczego:) i kiedy trzeba robić tego StringBuildera a kiedy nie. Generalnie zwykle IDE jest tak mądre, że podpowiada kiedy tak a kiedy nie.

Ogólnie IntelliJ to całkiem mądre IDE i zwykle warto zerknąć co tam podpowiada/propnuje jak coś podświetla bo może to być coś mądrego:) W tym przypadku po najechaniu na podświetlone += i wciśnięciu alt + enter mamy propozycje podmiany na StringBuilder'a.

![image](https://user-images.githubusercontent.com/18233796/37983405-abac0ffc-31f3-11e8-8bf7-6ac91a1b681c.png)